### ISSUE TITLE
Transition Errors

### ISSUE THREAD

**Issue by brentarias** [damianopetrungaro/generator-angular-rest#1]
After generating angular-rest with yo, I launched and saw a minimalist page that said:

>AngularJS-boilerplate
A micro AngularJS boilerplate for start projects with mocking and routing modules ready, based on John Papa's style guide

There are no buttons, menu options or anything else.  It is just bare bones.

The console output contained this:

'Error: transition superseded
    at v [as $get] (angular-ui-router.min.js:7)
    at Object.invoke (angular.js:5003)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:14516
(anonymous) @ angular.js:11004
g @ angular.js:16869
$digest @ angular.js:17982
$apply @ angular.js:18280
(anonymous) @ angular.js:1912
invoke @ angular.js:5003
c @ angular.js:1910
Pc @ angular.js:1930
ue @ angular.js:1815
(anonymous) @ angular.js:33340
b @ angular.js:3435
angular.js:14516 Error: transition prevented
    at v [as $get] (angular-ui-router.min.js:7)
    at Object.invoke (angular.js:5003)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:14516
(anonymous) @ angular.js:11004
g @ angular.js:16869
$digest @ angular.js:17982
$apply @ angular.js:18280
(anonymous) @ angular.js:1912
invoke @ angular.js:5003
c @ angular.js:1910
Pc @ angular.js:1930
ue @ angular.js:1815
(anonymous) @ angular.js:33340
b @ angular.js:3435
angular.js:14516 Error: transition aborted
    at v [as $get] (angular-ui-router.min.js:7)
    at Object.invoke (angular.js:5003)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:14516
(anonymous) @ angular.js:11004
g @ angular.js:16869
$digest @ angular.js:17982
$apply @ angular.js:18280
(anonymous) @ angular.js:1912
invoke @ angular.js:5003
c @ angular.js:1910
Pc @ angular.js:1930
ue @ angular.js:1815
(anonymous) @ angular.js:33340
b @ angular.js:3435
angular.js:14516 Error: transition failed
    at v [as $get] (angular-ui-router.min.js:7)
    at Object.invoke (angular.js:5003)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995)
    at angular.js:4795
    at d (angular.js:4944)
    at e (angular.js:4969)
    at Object.invoke (angular.js:4995) "Possibly unhandled rejection: {}"
(anonymous) @ angular.js:14516
(anonymous) @ angular.js:11004
g @ angular.js:16869
$digest @ angular.js:17982
$apply @ angular.js:18280
(anonymous) @ angular.js:1912
invoke @ angular.js:5003
c @ angular.js:1910
Pc @ angular.js:1930
ue @ angular.js:1815
(anonymous) @ angular.js:33340
b @ angular.js:3435
app.js:715 staticsHomeCtrl init
platform-implementation.js:100InboxSDK: Unsupported origin http://127.0.0.1:8080
DevTools failed to parse SourceMap: https://www.inboxsdk.com/build/platform-implementation.js.map`

I noticed the routes allow for users to be individually viewed.  So I manually typed this address:

>http://127.0.0.1:8080/src/users/1

That address brought up the following page content:

>User short view

The page did not actually show any user.  It only showed the aforementioned text.  The console output also gave a series of transition errors.

Any thoughts on what is causing this?

**Comment by damianopetrungaro** [comment]
Hey @brentarias !
I have never had this problem.
Today i'll try to see what happened :D

**Comment by damianopetrungaro** [comment]
The issue il related to this:

https://github.com/angular-ui/ui-router/issues/2889

For now i'll add a workaround:
```js
angular.module('app', ['core.routing', 'core.mocking', 'core.validator', 'ui.router'])
    // This config hide the rejection
    .config(['$qProvider', function ($qProvider) {
        $qProvider.errorOnUnhandledRejections(false);
    }]);

```

**Comment by damianopetrungaro** [comment]
Solved in v0.3.1
